Mobile Buy SDK/Mobile Buy SDK/Operations/BUYRequestOperation.m

```diff
@@ -144,9 +143,16 @@ - (NSURLSessionDataTask *)requestUsingPollingIfNeeded:(NSURLRequest *)request co
 		 * the polling process.
 		 */
 		if (self.pollingHandler && self.pollingHandler(json, response, error)) {
-			NSURLSessionDataTask *task = [self requestUsingPollingIfNeeded:request completion:completion];
-			self.runningTask = task;
-			[task resume];
+
+			dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(self.pollingInterval * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
+				if (self.isCancelled) {
```

Comment:
You're using the method name and the property syntax. Sure, it works for the getter, but try this with the setter and you'll regret it. (OK it doesn't apply here with a read-only property, but…) We should be consistent and use `cancelled` with property dot syntax.